packages/renderer/src/App.svelte

```diff
@@ -170,10 +171,12 @@ tablePersistence.storage = new PodmanDesktopStoragePersist();
             <ContainerDetails containerID={meta.params.id} />
           </Route>
         </Route>
-
         <Route path="/kube/play" breadcrumb="Podman Kube Play">
           <KubePlayYAML />
         </Route>
+        <Route path="/kube/apply" breadcrumb="Apply YAML">
```

Comment:
```suggestion
        <Route path="/kubernetes/apply" breadcrumb="Apply YAML">
```

I think it may be nice to use the `kubernetes` word for kubernetes related path; this is something we do for resources, and I think it is nice to scope them together

https://github.com/podman-desktop/podman-desktop/blob/0339c7c43689a645c3d313f5c2d356e20907b354/packages/renderer/src/App.svelte#L296